quantum/action.c

```diff
@@ -1099,6 +1099,10 @@ bool is_tap_key(keypos_t key) {
  * FIXME: Needs documentation.
  */
 bool is_tap_record(keyrecord_t *record) {
+    // layer_switch_get_action would attempt to read OOB for TICKs
+    if (IS_NOEVENT(record->event)) {
```

Comment:
May not be necessary anymore if action_tapping isn't led into the sequence of branches that triggered it after more fixes, but it is definitely possible.